pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/TableRebalancerAdminToolClusterIntegrationTest.java

```diff
@@ -1,871 +0,0 @@
-/**
```

Comment:
Are we testing the functionality of TableRebalancer in some other class? This class had some important tests for TableRebalance and tested the end to end path from pinot-admin script to all the way into TableRebalancer. Not sure if deleting this would be helpful

Reply:
The tests also covered increasing the number of replicas, decreasing the number of replicas, direct transition to target ideal state, incremental transition to ideal state and tests for realtime tables as well.

Reply:
The TableRebalancer functionalities are covered in:
- TableRebalancerTest: mainly focus on the calculation of the next state
- TableRebalancerClusterTest: covers the whole rebalance process
The offline/real-table cases are covered in:
- OfflineNonReplicaGroupSegmentAssignmentTest
- OfflineReplicaGroupSegmentAssignmentTest
- RealtimeNonReplicaGroupSegmentAssignmentTest
- RealtimeReplicaGroupSegmentAssignmentTest